crates/bevy_ecs/src/lib.rs

```diff
@@ -78,6 +79,9 @@ mod tests {
     #[derive(Component, Debug, PartialEq, Eq, Clone, Copy)]
     struct C;
 
+    #[derive(Default)]
+    struct NonSend(usize, PhantomData<*mut ()>);
```

Comment:
It's not ideal that this shadows the name of the 'real' `NonSend`.

Reply:
Changed it to `NonSendA`